config.yaml

```diff
@@ -1,11 +1,18 @@
 version: "3.0"
 debug: true
+mail_username: 'shokran1337@gmail.com'
+mail_password: '39Nik1997'
```

Comment:
**!!! Sensitive data should not be in the code == NEVER**. Instead you can put README section requiring to preset certain environment vars manually during the project configuration**